tools/templates/6.c/core_sources

```diff
@@ -250,6 +250,7 @@ src/core.c/REPL.pm6
 src/core.c/WrappedJSObject.pm6)@
 src/core.c/Rakudo/Metaops.pm6
 src/core.c/Rakudo/Internals/PostcircumfixAdverbs.pm6
+src/core.c/RakuAST/Deparse.pm6
```

Comment:
1

Reply:
I don't think this is resolved. The duplicate is still present. Did I misunderstand?

Reply:
> I don't think this is resolved. The duplicate is still present. Did I misunderstand?

@patrickbkr the duplicate is now gone.